app/src/sharedTest/java/org/oppia/android/app/topic/TopicFragmentTest.kt

```diff
@@ -214,8 +225,39 @@ class TopicFragmentTest {
     }
   }
 
+  @Test
+  fun testTopicFragment_practiceTabEnabled_practiceTopicTabIsDisplayedInTabLayout() {
+    initializeApplicationComponent(practiceTabIsEnabled = true)
+    launchTopicActivityIntent(internalProfileId, FRACTIONS_TOPIC_ID).use {
+      val practiceTab =
+        TopicTab.getTabForPosition(position = PRACTICE_TAB_POSITION, enablePracticeTab)
+      onView(withText(practiceTab.name)).check(
+        matches(
+          isDescendantOfA(
+            withId(
+              R.id.topic_tabs_container
+            )
+          )
+        )
+      )
+    }
+  }
+
+  @Test
+  fun testTopicFragment_practiceTabDisabled_practiceTopicTabIsNotDisplayedInTabLayout() {
```

Comment:
I think there should be tests to check that on configuration-change the tab remains hidden/visible.

Reply:
Nice idea. Done.